app/models/work_activity.rb

```diff
@@ -62,8 +64,17 @@ def comment_html
       Kramdown::Document.new(text).to_html
     end
 
-    # Returns the message formatted to display changes that were logged as an activity
-    def changes_html
+    # Returns the message formatted to display _file_ changes that were logged as an activity
+    def file_changes_html
+      changes = JSON.parse(message)
+      changes_html = changes.map do |change|
+        "<i>#{change['action']}</i> <span>#{change['filename']}<span><br/>"
+      end
+      "<p><b>Files</b>: #{changes_html.join}</p>"
```

Comment:
I wonder if making the changes something like a data table would be better for screen readers.  I know paragraphs of text are not the greatest.  This should not block this PR, but something to think about.

Reply:
Good idea, I added a separate ticket for that https://github.com/pulibrary/pdc_describe/issues/553